lib/ansible/modules/packaging/os/dnf.py

```diff
@@ -589,6 +582,12 @@ def _base(self, conf_file, disable_gpg_check, disablerepo, enablerepo, installro
         self._configure_base(base, conf_file, disable_gpg_check, installroot)
         self._specify_repositories(base, disablerepo, enablerepo)
         try:
+            try:
+                base.init_plugins(set(self.disable_plugin), set(self.enable_plugin))
+                base.pre_configure_plugins()
+                base.configure_plugins()
+            except AttributeError:
+                pass  # older versions of dnf didn't require this and don't have these methods
```

Comment:
Can these configuration methods raise ```dnf.exceptions.RepoError```?  If not, they should probably be above the RepoError try: except instead of nested inside of it.

Reply:
@abadger not to the best of my knowledge, no ... will update 👍